docs/_data/xpath_funs.yml

```diff
@@ -186,4 +186,18 @@ langs:
             description: "The number of children."
         examples:
           - code: '//Identifier[pmd-kotlin:hasChildren(3)]'
+            outcome: "true or false"
+
+      - name: hasImport
+        returnType: "xs:boolean"
+        shortDescription: "Test for an import header"
+        description: "Returns true if the document has the specified package in the import header"
+        parameters:
+          - name: "package"
+            type: "xs:string"
+            description: "The fully qualified package name"
+        examples:
+          - code: '/KotlinFile[pmd-kotlin:hasImport('javax.xml.xpath')]'
+            outcome: "true or false"
+          - code: '//FunctionBody//T-Identifier[@Text='newXPath' and pmd-kotlin:hasImport('javax.xml.xpath')]'
```

Comment:
it checks for the argument being contained in what is specified in the import statement, so it can include the class name. Not recommended, because you will miss the case when using a wildcard (*). 
Specifying a wildcard will not match, we don't support it in the function.
It believe better keep it simple, not regexp. Can use 'javax.xml' and normal logic (like 'or') to combine multiple hasImport function calls.

Reply:
check for wrong input, don't think that is really needed. What edge cases are you thinking of?